apps/datahub/src/app/record/custom-api/custom-api.component.html

```diff
@@ -0,0 +1,47 @@
+<div
+  class="bg-primary-opacity-10 pt-8 overflow-visible sm:pt-5"
+  style="height: 512px"
```

Comment:
Why a height ? Is it supposed to be scrollable ?
Otherwise the height is built from the content.

BTW, you should use `class="h-[512px]"` instead

Reply:
This is a WIP, the style is not done at all, I barely did the structure here ^^. This `div` was copied from the `data view` part of the record page (which yes, has a height for some reason), just to have a `div` that would go across the screen. I'm planning on doing styling after everything else.